### DISCUSSION TITLE
About Using prebuilt Image

### DISCUSSION THREAD

**Discussion by Fe-r-oz** [Xilinx/finn#856] [Q&A] [answered]
Dear Experts,

I hope this message finds you well.  These are the steps I followed before I found myself with a problem.

- Login to Docker
- docker pull maltanar/finn:dev_latest
- FINN_DOCKER_TAG=maltanar/finn:dev_latest
- FINN_DOCKER_PREBUILT=1
- docker run "image ID"

I am having a problem in launching docker prebuilt image. I have attached the snapshot representing the problem. 

![Screenshot_select-area_20230717115608](https://github.com/Xilinx/finn/assets/93876775/8cbe1e14-6fc2-4f4b-91e7-56ef1f809e54)

Since I am having trouble building the docker image, I hope to use this prebuild image instead. 
Kindly please assist me in running successfully the docker prebuilt image
**Thank you very much** for your time and consideration.

Sincerely,
Feroz Ahmad

**Comment by auphelia** [comment, marked as answer]
Hi @Fe-r-oz ,

Even with the prebuilt image, you should use the provided [run-docker.sh](https://github.com/Xilinx/finn/blob/main/run-docker.sh) script to setup all the dependencies. After setting the environmental variables, you can launch the container using the mentioned script. Please make sure you have also set the env vars to your Vivado/Vitis installation.

As a remark, [the prebuilt image support will be deprecated](https://github.com/Xilinx/finn/discussions/764) and thus `main_latest` already doesn't exist anymore. You can currently use `dev_latest`, like you have mentioned in your message. Please make sure to also work with your repository on `dev`, otherwise there might be compatibility issues.
